assets/script.js

```diff
@@ -0,0 +1,10 @@
+$(window).resize(function() {
+  console.log('resize called');
+  if ($(window).width() < 768) {
+    $("nav").addClass("navbar-fixed-top");
+  } else {
+    $("nav").removeClass("navbar-fixed-top");
+  }
+});
+
+resize();
```

Comment:
ill probably just inline this JS rather than have a whole separate file (which is another web request)